packages/ember-htmlbars/lib/env.js

```diff
@@ -2,35 +2,70 @@ import environment from "ember-metal/environment";
 
 import DOMHelper from "dom-helper";
 
-import inline from "ember-htmlbars/hooks/inline";
-import content from "ember-htmlbars/hooks/content";
-import component from "ember-htmlbars/hooks/component";
-import block from "ember-htmlbars/hooks/block";
-import element from "ember-htmlbars/hooks/element";
+import { hooks } from "htmlbars-runtime";
+import merge from "ember-metal/merge";
+
 import subexpr from "ember-htmlbars/hooks/subexpr";
-import attribute from "ember-htmlbars/hooks/attribute";
 import concat from "ember-htmlbars/hooks/concat";
-import get from "ember-htmlbars/hooks/get";
-import set from "ember-htmlbars/hooks/set";
+import linkRenderNode from "ember-htmlbars/hooks/link-render-node";
+import createFreshScope from "ember-htmlbars/hooks/create-fresh-scope";
+import createShadowScope from "ember-htmlbars/hooks/create-shadow-scope";
+import bindSelf from "ember-htmlbars/hooks/bind-self";
+import bindScope from "ember-htmlbars/hooks/bind-scope";
+import bindLocal from "ember-htmlbars/hooks/bind-local";
+import getRoot from "ember-htmlbars/hooks/get-root";
+import getChild from "ember-htmlbars/hooks/get-child";
+import getValue from "ember-htmlbars/hooks/get-value";
+import cleanup from "ember-htmlbars/hooks/cleanup";
+import classify from "ember-htmlbars/hooks/classify";
+import component from "ember-htmlbars/hooks/component";
+import lookupHelper from "ember-htmlbars/hooks/lookup-helper";
+import hasHelper from "ember-htmlbars/hooks/has-helper";
 
 import helpers from "ember-htmlbars/helpers";
 
-export default {
-  hooks: {
-    get: get,
-    set: set,
-    inline: inline,
-    content: content,
-    block: block,
-    element: element,
-    subexpr: subexpr,
-    component: component,
-    attribute: attribute,
-    concat: concat
-  },
+var emberHooks = merge({}, hooks);
 
-  helpers: helpers,
+merge(emberHooks, {
+  linkRenderNode: linkRenderNode,
+  createFreshScope: createFreshScope,
+  createShadowScope: createShadowScope,
+  bindSelf: bindSelf,
+  bindScope: bindScope,
+  bindLocal: bindLocal,
+  getRoot: getRoot,
+  getChild: getChild,
+  getValue: getValue,
+  subexpr: subexpr,
+  concat: concat,
+  cleanup: cleanup,
+  classify: classify,
+  component: component,
+  lookupHelper: lookupHelper,
+  hasHelper: hasHelper
+});
 
+import debuggerKeyword from "ember-htmlbars/keywords/debugger";
+import withKeyword from "ember-htmlbars/keywords/with";
+import outlet from "ember-htmlbars/keywords/outlet";
+import unbound from "ember-htmlbars/keywords/unbound";
+import view from "ember-htmlbars/keywords/view";
+import componentKeyword from "ember-htmlbars/keywords/component";
+import partial from "ember-htmlbars/keywords/partial";
+
+merge(emberHooks.keywords, {
```

Comment:
Currently there are no upstream keywords in HTMLBars, so I believe this needs to be:
```
emberHooks.keywords = {};
merge(emberHooks.keywords, {...});
````